english/destiny/destiny.md

```diff
@@ -22,14 +22,14 @@ and you might be someone [who knocks](../say-my-name/say-my-name.md).
 
 or perhaps you are someone [who opens](https://www.biblegateway.com/passage/?search=Revelation+3%3A20).
 
-If you have finished pursuing your destiny, it may be time for a 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.